data/scripts/variants_spdi.py

```diff
@@ -0,0 +1,124 @@
+import gzip
+import csv
+import hashlib
+import os
+from ga4gh.vrs.extras.translator import Translator
+from ga4gh.vrs.dataproxy import create_dataproxy
+
+chr_map = {
+    '1': 'NC_000001.11',
+    '2': 'NC_000002.12',
+    '3': 'NC_000003.12',
+    '4': 'NC_000004.12',
+    '5': 'NC_000005.10',
+    '6': 'NC_000006.12',
+    '7': 'NC_000007.14',
+    '8': 'NC_000008.11',
+    '9': 'NC_000009.12',
+    '10': 'NC_000010.11',
+    '11': 'NC_000011.10',
+    '12': 'NC_000012.12',
+    '13': 'NC_000013.11',
+    '14': 'NC_000014.9',
+    '15': 'NC_000015.10',
+    '16': 'NC_000016.10',
+    '17': 'NC_000017.11',
+    '18': 'NC_000018.10',
+    '19': 'NC_000019.10',
+    '20': 'NC_000020.11',
+    '21': 'NC_000021.9',
+    '22': 'NC_000022.11',
```

Comment:
I downloaded one of the vcf file, seems they have a bad naming for chromosome 22: ##contig=<ID=22>. You might want to adjust that key mapping here.